### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by xdamman**
Please merge this pull request to activate your Open Collective https://opencollective.com/strider

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,6 +91,7 @@
     "mongoose": "^4.7.6",
     "morgan-debug": "^1.0.0",
     "node-rsa": "^0.4.0",
+    "opencollective": "^1.0.0",
```
Comment (knownasilya) [inline package.json]: What does this do?

File: README.md [hunk 2]
```diff
@@ -180,6 +182,78 @@ To build the documentation run `npm run docs` and the documentation will be acce
 
 **[View Strider API Docs](http://strider-api-docs.surge.sh/)**
 
+## Backers
+
+Support us with a monthly donation and help us continue our activities. [[Become a backer](https://opencollective.com/strider#backer)]
+
+<a href="https://opencollective.com/strider/backer/0/website" target="_blank"><img src="https://opencollective.com/strider/backer/0/avatar.svg"></a>
```
Comment (knownasilya) [inline README.md]: Not a fan of the mass of links. Is there an alternative single image option?

File: package.json (hunk 1 above)
Comment (xdamman) [inline package.json]: It's used by the postinstall script to show a please donate message. Try it!

File: README.md (hunk 2 above)
Comment (xdamman) [inline README.md]: They all resolve into 1px transparent png if there is no backer to show (so it doesn't pollute visually for the end user, it's just annoying in the markdown version). It's sadly the only way we can make each backer link to their own profile.

**Pull request merged**